brave-core/src/test/java/com/github/kristofa/brave/ITBrave.java

```diff
@@ -107,7 +108,11 @@ public Integer call() throws Exception {
             assertTrue(localSpan.id != 0);
             assertEquals(localSpanName, localSpan.name);
 
-            assertEquals("Span ids should be different.", 3, Stream.of(serverSpan.id, clientSpan.id, localSpan.id).distinct().count());
```

Comment:
no, but this is kind of tricky with maven and IDEs. I thought I could sacrifice the readability of this test